vpr/src/place/RL_agent_util.h

```diff
@@ -30,10 +30,4 @@ void assign_current_move_generator(std::unique_ptr<MoveGenerator>& move_generato
  * @ brief move the updated current_move_generator to its original move_Generator structure based on he placer_options and the agent state
  */
 void update_move_generator(std::unique_ptr<MoveGenerator>& move_generator, std::unique_ptr<MoveGenerator>& move_generator2, e_agent_state agent_state, const t_placer_opts& placer_opts, bool in_quench, std::unique_ptr<MoveGenerator>& current_move_generator);
```

Comment:
I thought it would be safer to perform the mapping inside the agent itself because both agent and logical block types were using the same type (t_logical_block_type), which was confusing. KArmedBanditAgent constructor initializes a vector that holds the mapping from agent block types to logical block types and all type conversion are done internally.

Reply:
Yes, that makes sense and once I got that far in the code I saw what you had done. Seems like a good change.